tc/core/cuda/cuda_rtc.h

```diff
@@ -57,13 +57,15 @@ class CudaRTCFunction {
       std::vector<long> params,
       std::vector<void*> outputs,
       std::vector<const void*> inputs,
+      uint32_t timeout,
```

Comment:
definitely not, we JIT compile kernels, let's hardcode the timeouts instead of increasing the complexity of the flow